Makefile

```diff
@@ -209,6 +215,13 @@ setup-integration-local: setup-integration-local
 	gcloud -q container images add-tag \
 		gcr.io/kritis-tutorial/nginx-no-digest:latest \
 		gcr.io/$(GCP_PROJECT)/nginx-no-digest:latest
+	gcloud projects add-iam-policy-binding ${GCP_PROJECT} \
+		--member=serviceAccount:kritis-ca-admin@${GCP_PROJECT}.iam.gserviceaccount.com \
+		--role=roles/containeranalysis.notes.occurrences.viewer
+	gcloud projects add-iam-policy-binding ${GCP_PROJECT} \
+		--member=serviceAccount:kritis-ca-admin@${GCP_PROJECT}.iam.gserviceaccount.com \
+		--role=roles/containeranalysis.occurrences.viewer
```

Comment:
`https://github.com/grafeas/kritis/blob/master/docs/install.md` also lists `editor` permissions, and `notes.viewer`. Not needed?

Reply:
Sigh... I think occurrences.editor is kindof needed, because ISP creates attestations.  But this behavior (caching ISP decisions as attestations) is not really exercised in the test, so in reality the permission is not required.  I added it anyway...

I don't think note editing is needed though.

Reply:
No, the setup script does that, it's a one-time thing.